0.download_data/README.md

```diff
@@ -0,0 +1,11 @@
+# Download Data
+
+In this module, we present our method for downloading nucleus morphology data.
+
+## Download/Process Data
+
+Complete instructions for data download and processing can be found at: https://github.com/WayScience/mitocheck_data
+
+## Usage
+
+In this repository, all data is accessed from a version of [mitocheck_data](https://github.com/WayScience/mitocheck_data) and used to create training_data.csv.gz.
```

Comment:
It will be good to specify the version here: (I think it is this hash: de21b9c3201ba4298db2b1704f3ae510a5dc47e2)

Better yet, it's actually better to only specify the version once (so that you don't have to update it in multiple places).

So, what you have here is good, except you should explicitly note that your training data uses version controlled data from the mitocheck repo (and then point to where in the training notebook)

Reply:
The detail that you have in the README in the next module is perfect :chef_kiss:

Reply:
Resolved in https://github.com/WayScience/phenotypic_profiling_model/pull/1/commits/87cdcff16316b3c6dbb424b493c1b7573850ed6a